pytest_fixtures/component/provision_pxe.py

```diff
@@ -155,6 +196,21 @@ def module_provisioning_sat(
     return Box(sat=sat, domain=domain, subnet=subnet)
 
 
+@pytest.fixture(scope='module')
+def module_provisioning_sat_pull_provider(module_provisioning_sat):
+    """
+    This fixture enables pull provider rex mode on Satellite set up
+    for PXE provisioning
```

Comment:
It was hard for me to parse this sentence :D
```suggestion
    This fixture sets REX mode to pull
```